graphdatascience/tests/integration/test_training_pipeline.py

```diff
@@ -211,6 +211,20 @@ def test_add_random_forest_lp_pipeline(lp_pipe: LPTrainingPipeline) -> None:
     assert rf_parameter_space[0]["maxDepth"] == 1337
 
 
+@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 2, 0))
+def test_add_mlp_with_wrong_config_lp_pipeline(lp_pipe: LPTrainingPipeline) -> None:
+    with pytest.raises(Exception):
+        lp_pipe.addMLP(hiddenLayerSizes=64)
```

Comment:
nit: this only tests server side validation. the above for RF tested client side logic